Server/src/main/java/app/ModelController.java

```diff
@@ -102,7 +102,7 @@ public TrainedModel getModelForAssetType(List<TrainedModel> trainedModels, Strin
             if (tm.getAssetTypeID() == Integer.parseInt(assetTypeID))
                 return tm;
         }
-        trainedModels.add(modelDAOImpl.getModelsByAssetTypeID(assetTypeID));
+        trainedModels.add(modelDAOImpl.getModelsByAssetTypeID(assetTypeID, 1));
```

Comment:
I believe this would be a place to be using the constants defined for the status_id in the Constants file

Reply:
yea, added the constant after, forgot to check, thanks